src/bench/bench.cpp

```diff
@@ -69,6 +69,13 @@ void benchmark::BenchRunner::RunAll(const Args& args)
         }
 
         Bench bench;
+        if (args.one_iteration) {
+            ankerl::nanobench::Config config;
+            config.mNumEpochs = 1;
```

Comment:
Could also add separate flags for `--num-epochs` and `--epoch-iterations` (which default to `11` and `0` respectively).

Reply:
I'd use `bench.epochs(1).epochIterations(1)` instead of modifying the `Config` members directly

Reply:
That's a great suggestion, will do.